modules/message-lane/src/lib.rs

```diff
@@ -194,8 +194,14 @@ decl_module! {
 		pub fn set_owner(origin, new_owner: Option<T::AccountId>) {
 			ensure_owner_or_root::<T, I>(origin)?;
 			match new_owner {
-				Some(new_owner) => ModuleOwner::<T, I>::put(new_owner),
-				None => ModuleOwner::<T, I>::kill(),
+				Some(new_owner) => {
+					ModuleOwner::<T, I>::put(&new_owner);
+					frame_support::debug::info!(target: "runtime", "Setting pallet Owner to: {:?}", new_owner);
```

Comment:
Why is the `target` set to `runtime`? Why do we need `target` at all?

Reply:
That's what everything else in the `message-lane` pallet uses. I didn't want to change all the logging in this PR.